### PR TITLE
Update making contributions pages

### PR DESCRIPTION
**Pull request by line47**

### CODE REVIEW

File: _data/theme.yml [hunk 1]
```diff
@@ -63,12 +63,12 @@ colors:
...
-    primary-footer-link: 'black'
-    footer-bg: 'DarkSlateGray'
-    footer-text: 'white'
-    footer-link: 'LightBlue'
+  # footer:
+  #   primary-footer-bg: 'Gainsboro'
+  #   primary-footer-link: 'black'
+  #   footer-bg: 'DarkSlateGray'
+  #   footer-text: 'white'
+  #   footer-link: 'LightBlue'
```
Comment (line47) [inline _data/theme.yml]: commented this out as your site has a custom footer

File: _sass/components/_variables.scss [hunk 2]
```diff
@@ -35,27 +35,3 @@ $spacing-small: 1rem;
...
-// $h2: $h2-font-size;
-// $h3: $h3-font-size;
-// $h4: $h4-font-size;
-// $h5: $h5-font-size;
-// $h6: $h6-font-size;
-//
-// $space-1: $spacing-small;
-// $space-2: $spacing-md-small;
-// $space-3: $spacing-medium;
-// $space-4: $spacing-large;
```
Comment (line47) [inline _sass/components/_variables.scss]: Removed this as your site is no longer using basscss

File: pages/making-contributions/contribution-limits.md [hunk 3]
```diff
@@ -1,18 +1,13 @@
...
-
 sidenav: manage
-
 scripts:
-
 permalink: /making-contributions/contribution-limits/
 ---
 
-## What are the contribution limits?
+# What are the contribution limits?
```
Comment (line47) [inline pages/making-contributions/contribution-limits.md]: Updating headers so an H1 is the first heading on the page, if the size is too big we can change that with styles 👍

File: pages/making-contributions/traditional-and-roth.md [hunk 4]
```diff
@@ -1,24 +1,34 @@
...
 title: Making contributions
-
 styles:
-
 sidenav: manage
-
 scripts:
-
 permalink: /making-contributions/traditional-and-roth/
-
```
Comment (line47) [inline pages/making-contributions/traditional-and-roth.md]: Closed the empty lines in the front matter on these files

File: pages/making-contributions/traditional-and-roth.md [hunk 5]
```diff
@@ -1,24 +1,34 @@
...
+## Traditional contributions
+
+Traditional contributions
+are made before taxes are
+taken out, which means you’ll
+pay taxes when you withdraw
+your money.
+  </div>
+  <div class="usa-width-one-half" markdown="1">
+## Roth contributions
```
Comment (line47) [inline pages/making-contributions/traditional-and-roth.md]: note the `markdown="1"` in the HTML which allows the use of markdown inside the `Div`

File: pages/makingContributions-03.md [hunk 6]
```diff
@@ -1,69 +0,0 @@
----
```
Comment (line47) [inline pages/makingContributions-03.md]: This page was a duplicate. Removed

File: pages/makingContributions.md [hunk 7]
```diff
@@ -1,140 +0,0 @@
----
```
Comment (line47) [inline pages/makingContributions.md]: This page was also a duplicate

File: pages/making-contributions/traditional-and-roth.md (hunk 5 above)
Comment (chachi69) [reply pages/making-contributions/traditional-and-roth.md]: I was wondering about that. Excellent!

**Pull request merged**